masterlist.yaml

```
@@ -7722,6 +7722,75 @@ plugins:
         crc: 0x68A235DA
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 6
+       
+  - name: 'AncientLand2.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/22900'
```

Can you please add a trailing slash to the link? We do that for convenience & consistency:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/22900/'
```